.github/workflows/reusable_compatibility.yml

```diff
@@ -36,61 +36,66 @@ jobs:
       working-directory: ${{github.workspace}}/tag_version
       run: .github/scripts/install_hwloc.sh
 
-    - name: Configure "tag" UMF build
-      working-directory: ${{github.workspace}}/tag_version
+    - name: Checkout latest UMF version
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```

Comment:
Please, use the latest v4.2.2 instead

Reply:
done - note that we still use 4.1.1 in other places in our code

Reply:
Right, there shall be some bump made some day for that. There would be just less work in the aforementioned bump if we already were using the latest versions of actions when we add new code using them.